lib/mbta_v3_api/repository_cache/key_generator.ex

```diff
@@ -0,0 +1,22 @@
+defmodule MBTAV3API.RepositoryCache.KeyGenerator do
```

Comment:
Since this module won't be referenced directly outside of `RepositoryCache`, it might make sense to define it as a nested module directly within `RepositoryCache` instead of giving it its own file and subdirectory.

Reply:
Oh, what I meant was
```elixir
defmodule MBTAV3API.RepositoryCache do
  defmodule KeyGenerator do
  end
end
```
but this seems reasonable too.